definitions/checks/unread_mail.rb

```diff
@@ -0,0 +1,40 @@
+class Checks::UnreadMail < ForemanMaintain::Check
+  metadata do
+    label :unread_mail
+    description 'Check for unread mail for root user.'
+    tags :default
+  end
+
+  attr_reader :file_path
+
+  def initialize
+    @file_path = '/var/spool/mail/root'
```

Comment:
@gnurag,
Thank you for providing information. 
I will go through it and fix the code as per your suggestions.